plugins/grid/wayfire/plugins/grid.hpp

```diff
@@ -38,6 +26,24 @@ enum slot_t
     SLOT_TR     = 9,
 };
 
+/**
+ * name: request
+ * on: core
+ * when: Emitted before move renders a grid indicator and sets the slot.
+ * carried_out: true if a plugin can handle move request to grid.
+ */
+struct grid_request_signal
+{
+    /* True if a plugin handled this signal */
+    bool carried_out = false;
+    bool force_off = false;
+    wf::output_t *output;
+    wf::point_t input;
```

Comment:
This one would need a docstring to clarify what it is actually (something like "input position relative to the indicated output")